extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php

```diff
@@ -12,41 +12,118 @@ class NodeImage extends Node {
 
 	public function getData() {
 		if ( !isset( $this->data ) ) {
-			$imageData = $this->getRawValueWithDefault( $this->xmlNode );
-
-			if( is_string($imageData) && PortableInfoboxDataBag::getInstance()->getGallery($imageData)) {
-				$imageData = PortableInfoboxDataBag::getInstance()->getGallery($imageData);
+			$this->data = array();
+
+			$parameterValue = $this->getRawValueWithDefault( $this->xmlNode );
+
+			if ( $this->containsTabberOrGallery( $parameterValue ) ) {
+				$parsed = $this->getExternalParser()->parseRecursive( $parameterValue );
+
+				$items = $this->getGalleryItems( $parsed ) + $this->getTabberItems( $parsed );
```

Comment:
If the intent of this line is to merge arrays resulting from getGalleryItems with those from getTabberItems, the '+' operator will yield different results unless list indexes are unique. Is it intended?